.changeset/empty-pens-listen.md

```diff
@@ -0,0 +1,5 @@
+---
+"@equinor/mad-components": patch
+---
+
+style: swapped white with variable color
```

Comment:
Good to see that you added a changeset! Since this is user facing, try renaming this to something like "Set the `Dialog` container color to be dynamic." or something like that

Reply:
fixed in dc0661